app/FAIRshakeWeb/view.py

```diff
@@ -43,8 +55,129 @@ def register():
     current_user={},
   )
 
-@app.route('/login', methods=['GET'])
-def login():
-  return render_template('login.html',
+@app.route('/project/<int:project>/resources', methods=['GET'])
+def resources(repository: RepositoryAPI, assessment: AssessmentAPI, score: ScoreAPI, project):
+  resources=repository.get(tag=project)
+  current_user_assessments = []
+  assessment_count = {}
+  aggregate_scores = {}
+  for resource in resources: #resource-id=1-7
+    try:
+      assessment.get(object=resource.id, user=current_user['id']) ### redesign, if there is an assessment for this object for this user
+      current_user_assessments.append(resource.id) # just array with resource ids
+    except:
+      pass
+    assessment_count[resource.id] = len(assessment.get(object=resource.id)) # all assessments from all users for all rubrics
```

Comment:
Do we really need the total assessment count for a given object or are we looking for the user assessment count? If so, can we obtain the count in the template instead having the full query already available?

Reply:
Want total assessment count. Don't have the query results either, I just used to check for an exception.